molecule/default/INSTALL.rst

```diff
@@ -0,0 +1,22 @@
+*******
```

Comment:
It is, I just missed this one when adding the new scenario. I think it might be trivial to make molecule use a different template dir than the one baked into molecule itself, so using `molecule init scenario` like I did here wouldn't result in too many surprises like this.

Reply:
File removed.